code/modules/clothing/suits/cloaks.dm

```diff
@@ -92,3 +93,4 @@
 	heat_protection = HEAD
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+	snug_fit = TRUE
```

Comment:
Imagine an atmos plasma flooding and just throwing a hat at you and you start receiving damage from fire, no.